tests/components/tasmota/test_sensor.py

```diff
@@ -490,7 +486,7 @@ async def test_status_sensor_state_via_mqtt(hass, mqtt_mock, setup_tasmota):
 
     # Pre-enable the status sensor
     entity_reg.async_get_or_create(
-        sensor.DOMAIN,
+        DOMAIN,
```

Comment:
Could you replace `sensor.DOMAIN` with `Platform.SENSOR` instead (and maybe do the other platforms at the same time - or keep for a separate PR)?